### DISCUSSION TITLE
Is there any plans for gpu support?

### DISCUSSION THREAD

**Discussion by jangjunho7** [klauspost/reedsolomon#290] [Ideas/Features]
Dear.

We have a plan to accelerate the reedsolomon code through gpu.
Do you have any support or plans for that feature?

Best regards.

**Comment by klauspost** [comment]
No. It is pointless. 

CPUs can easily outperform memory bandwidth, so moving data to/from GPU will by itself be slower than CPU computing.

**Reply by jangjunho7** [reply]
Oh, I see. Thank you for your answer
